clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/ActiveActiveStoreIngestionTask.java

```diff
@@ -592,6 +592,9 @@ private PubSubMessageProcessedResult processActiveActiveMessage(
               null,
               (schemaId) -> storeDeserializerCache.getDeserializer(schemaId, schemaId)));
     } else {
+      if (rmdWithValueSchemaID != null) {
```

Comment:
Could you add a small comment about this metric that it is only counting the update that will be produced to version topic? Because it is a bit confusing for other people what this is for.